src/lib/mongoBackend/collectionOperations.h

```diff
@@ -0,0 +1,47 @@
+#ifndef COLLECTIONOPERATIONS_H
```

Comment:
COLLECTIONOPERATIONS_H is not according to the (still unwritten) style guide.
Actually, this is not even mentioned yet in the 'style guide work in progress' ...

This part comes from the Google style guide (the Python script) and it states that the 
definition has to be SRC_LIB_MONGOBACKEND_COLLECTIONOPERATIONS_H_

Also, it must be present at the #endif, like this (note TWO spaces before '//'):

    #endif  // SRC_LIB_MONGOBACKEND_COLLECTIONOPERATIONS_H_

Not my idea - googles. But it IS a good idea to include the directories and not only the file name in the name of the definition.